optuna/samplers/_cmaes.py

```diff
@@ -131,22 +147,39 @@ def __init__(
         self._cma_rng = np.random.RandomState(seed)
         self._search_space = optuna.samplers.IntersectionSearchSpace()
         self._consider_pruned_trials = consider_pruned_trials
+        self._restart_strategy = restart_strategy
+        self._inc_popsize = inc_popsize
+        self._n_restarts = 0
 
         if self._consider_pruned_trials:
             self._raise_experimental_warning_for_consider_pruned_trials()
 
+        # TODO(c-bata): Support BIPOP-CMA-ES.
```

Comment:
Refs: https://github.com/CyberAgent/cmaes/blob/master/examples/bipop_cmaes.py

Reply:
I implemented IPOP-CMA-ES and BIPOP-CMA-ES on Goptuna. The benchmark results are below.

![rastrigin-function-dim-2-1a14f4610cabf55e51b3b006bfe723dcadef3a57d9eb5c82a09c2c9fbb804c42](https://user-images.githubusercontent.com/5564044/88660663-23e44380-d112-11ea-9d81-37620c419265.png)